man/manual.md

```diff
@@ -877,18 +910,19 @@ The output screen says:
        --source-dir=SOURCE_DIR
                        Base directory where source code files can be found.
 
-An example usage:
+Example usage:
 
     ./cppcheck gui/test.cpp --xml 2> err.xml
     htmlreport/cppcheck-htmlreport --file=err.xml --report-dir=test1 --source-dir=.
 
 # Bug hunting
 
-If you want to detect most bugs and can accept false alarms then Cppcheck has analysis for that.
+If you want to detect most bugs and can accept false alarms, then Cppcheck has analysis for that.
 
-This analysis is "soundy"; it should diagnose most bugs reported in CVEs and from dynamic analysis.
+This analysis is verbose; it should diagnose most bugs reported in CVEs and from dynamic analysis.
```

Comment:
actually "soundy" is a term that is used often.. you can google "soundy analysis" and get related hits.